src/jvmMain/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvWriter.kt

```diff
@@ -51,7 +51,7 @@ actual class CsvWriter actual constructor(
         writer.use { it.write() }
     }
 
-    fun writeString(write: ICsvFileWriter.() -> Unit): String {
+    internal fun writeAsString(write: ICsvFileWriter.() -> Unit): String {
```

Comment:
Hide this method for kotlin-csv users to put the `internal` modifier, because there didn't seem to be many use cases for using this method.